ext/dapr-ext-workflow/dapr/ext/workflow/dapr_workflow_client.py

```diff
@@ -37,9 +38,14 @@ class DaprWorkflowClient:
        This client is intended to be used by workflow application, not by general purpose
        application.
     """
-    def __init__(self, host: Optional[str] = None, port: Optional[str] = None):
+    def __init__(self, host: Optional[str] = None, port: Optional[str] = None, apiToken: Optional[str] = None):
```

Comment:
```suggestion
    def __init__(self, host: Optional[str] = None, port: Optional[str] = None, api_token: Optional[str] = None):
```

Python requires snake case instead of camelcase per style guide